app/abilities/photo.js

```diff
@@ -8,7 +8,7 @@ export default class Photo extends Ability {
   get canShowPhotoComments() {
     return (
       this.session.hasPermission('photo-comment.read') ||
-      this.model.photoAlbum.publiclyVisible
+      this.model.photoAlbum.get('publiclyVisible')
```

Comment:
Ember threw a very specific error about this that it is required because this object is a "special Ember object" "proxy".

Reply:
yeah I think this is because ember might not be able to guarantee that the attribute `publiclyVisible` is in store for the photoAlbum, because the publiclyVisible is a property on a property of `this.model`. It might need to fetch stuff for the photoAlbum, which means async stuff is going on. I never fully understood why it provides a proxy object instead of a promise though. 
tldr: this is fine, as long as it works.